src/pricing/cc-pricing-product-consumption.smart.js

```diff
@@ -56,12 +56,13 @@ defineComponent({
 
       product_lp.error$.subscribe(console.error),
       product_lp.error$.subscribe(() => (component.error = true)),
-      product_lp.value$.subscribe((product) => {
-        component.sections = product.sections;
-      }),
+      product_lp.value$.subscribe((product) => (component.sections = product.sections)),
 
       context$.subscribe(({ productId, zoneId, currency }) => {
 
+        // TODO, we may need to refine this reset
+        component.sections = null;
+
```

Comment:
At some point, we need to reset the `error` property too
`component.error = false;`

Should we do that here or at line 59?